### PR TITLE
Feat login modal

### LINKED ISSUE #18: create login/signin modal

### PR DESCRIPTION
**Pull request by Merdi-kim**
fixes #18 
Screenshot 2023-06-27 at 16 26 09

### CODE REVIEW

File: src/app/home/page.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+			<div className="w-10/12">
+				<Navbar  closeModal={setCloseModal} />
+				<HeroBanner />
+			</div>
+			{!closeModal && <LoginModal closeModal={setCloseModal}/>}
+		</div>
+	);
+};
+
+export default Home;
```
Comment (Jonath-z) [inline src/app/home/page.tsx:21]: This component is duplicated, you've implemented it into many PRs, to make it easy ... you can just create  it own PR that  can be quickly merged

File: src/components/HeroBanner.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+							/>
+						</Link>
+					))}
+				</div>
+			</div>
+		</div>
+	);
+};
+
+export default HeroBanner;
```
Comment (Jonath-z) [inline src/components/HeroBanner.tsx:47]: This one as well

**Review by Jonath-z — CHANGES_REQUESTED** [review]
A PR should gave a single implementation, this one implement so many thing than a login pop

File: src/app/home/page.tsx (hunk 1 above)
Comment (Merdi-kim) [reply src/app/home/page.tsx:21]: Which component are you talking about exactly ?

File: src/components/HeroBanner.tsx (hunk 2 above)
Comment (Merdi-kim) [reply src/components/HeroBanner.tsx:47]: I am not understanding what you mean by "component is duplicated"

File: src/app/home/page.tsx (hunk 1 above)
Comment (Jonath-z) [reply src/app/home/page.tsx:21]: I meant this selected component was implemented in an other PR again

File: src/components/HeroBanner.tsx (hunk 2 above)
Comment (Jonath-z) [reply src/components/HeroBanner.tsx:47]: This one also

**Review by Jonath-z — APPROVED** [review]
Need to resolve conflicts here

**Comment by Merdi-kim** [comment]
> A PR should gave a single implementation, this one implement so many thing than a login pop

Hey @Jonath-z , I would like to understand what you meant by this comment. Imo, this PR only implemented one thing, the login modal.

**Comment by Merdi-kim** [comment]
> Need to resolve conflicts here

This branch had no conflict with the main. It was the other branch with conflicts

**Pull request merged**